### PR TITLE
Feature/support numpy style

### PR DESCRIPTION
**Pull request by cbillingham**

### CODE REVIEW

File: doc/source/intro.rst [hunk 1]
```diff
@@ -272,3 +272,19 @@ use_types
...
+Numpy docstring output configuration options.
+
+use_keyword_section
+  Use **Keyword Arguments** section for all keyword-arguments. Defaults to
+  False. If set to False, all keyword-arguments will be placed under
+  **Parameters** section.
+
+use_types
+  Use types in argument output. Defaults to True. If False, argument,
+  keyword-argument, and attribute type definitions will be skipped.
```
Comment (AWhetter) [inline doc/source/intro.rst:288]: Trivial: And return type definitions as well?

File: doc/source/intro.rst [hunk 2]
```diff
@@ -272,3 +272,19 @@ use_types
...
+  This could be turned False for Python3, where Sphinx recognizes
```
Comment (AWhetter) [inline doc/source/intro.rst]: Typo: Python 3

**Review by AWhetter — APPROVED** [review]

File: doc/source/intro.rst (hunk 1 above)
Comment (cbillingham) [reply doc/source/intro.rst:288]: Hmmm... actually I don't think we skip return types at the moment. I could add that in pretty easily though I think.

Comment (cbillingham) [reply doc/source/intro.rst:288]: I'll make an issue for it Issue #4

**Pull request merged**